doc/specs/vulkan/chapters/fundamentals.txt

```diff
@@ -286,8 +286,8 @@ objects, and not to destroy them while they are still in use.
 
 Application-owned memory is immediately consumed by any Vulkan command it is
 passed into.
-The application can: alter or free this memory as soon as the commands that
-consume it have returned.
+The application must: not write to nor free this memory before the command
+that consumed it have returned.
```

Comment:
What's the reasoning for this change? This is practically how *all* functions work in any API, doesn't feel like it needs explicitly stating. The original statement was to clarify that pointers are not held onto - your new statement sort of re-iterates that you can't do something you can't really do in any C API, removing the original intent. I'd be ok if found a way to express both notions though, I just don't want to lose the original intent.

Reply:
BTW, I corrected this one. I kept the original intention except that I felt the need to replace "consumed" (which usually means the exact opposite of what it was used to describe here) with "borrow" (borrowed from Rust-like terminology).

I moved most of the threading intentions into the Threading chapter.
I moved to a more detailed description. I have added a strong const guarantee (though some layers like `unique_handles` could benefit from lesser guarantee).